src/main/webapp/app/pages/curation/button/SomaticGermlineToggleButton.tsx

```diff
@@ -26,16 +26,19 @@ function SomaticGermlineToggleButton({ hugoSymbol, firebaseDb, createGene }: ISo
   const newVariantType = isSomatic ? GERMLINE_PATH : SOMATIC_PATH;
 
   async function handleToggle() {
+    if (!firebaseDb) {
+      return;
+    }
     if (hugoSymbol) {
       // On curation page
       const genePath = getFirebasePath(isSomatic ? 'GENE' : 'GERMLINE_GENE', hugoSymbol);
       try {
         const snapshot = await get(ref(firebaseDb, genePath));
         if (!snapshot.exists()) {
-          await createGene(hugoSymbol, !isSomatic);
+          await createGene?.(hugoSymbol, !isSomatic);
         }
       } catch (error) {
-        notifyError(error);
+        notifyError(error as Error);
```

Comment:
Wonder if we want to just pass error as `unknown` (without casting) and then do a type check in `notifyError` to narrow the type since we do this a lot